packages/component/package.json

```diff
@@ -3,7 +3,14 @@
   "description": "React code splitting made easy.",
   "version": "5.15.3",
   "main": "dist/loadable.cjs.js",
```

Comment:
```suggestion
  "main": "./dist/loadable.cjs.js",
```

Could change just to make it uniform :D But harmless.

Reply:
Sure thing @wojtekmaj 😃